distribution/docs/src/main/resources/_contents/_security-framework/security-sts-ldap-login-contents.adoc

```diff
@@ -3,20 +3,20 @@
 
 The ${ddf-security} STS LDAP Login bundle enables functionality within the STS that allows it to use an LDAP to perform authentication when passed a `UsernameToken` in a `RequestSecurityToken` SOAP request.
 
-===== Installing the Security STS LDAP Claims Handler
+===== Installing the Security STS LDAP Login
 
 This bundle is not installed by default but can be added by installing the `security-sts-ldaplogin` feature.
 
-===== Configuring the Security STS LDAP Claims Handler
+===== Configuring the Security STS LDAP Login
 
-Configure the Security STS LDAP Claims Handler from the ${admin-console}:
+Configure the Security STS LDAP Login from the ${admin-console}:
 
 . Navigate to the ${admin-console}.
 . Select *${ddf-security} Application*.
```

Comment:
I know this isn't part of your changes, but can you change the `*` (bold) to only be around the `${ddf-security}` and not `Application`?